### ISSUE TITLE
Migrate supplemental repositories to github

### ISSUE THREAD

**Issue by matthewturk** [yt-project/bb-migration#6]
All the other repos will need to move to github as well, once we have a conversion process in place.

- [ ] ytep
- [x] website
- [x] ythub-docs
- [x] fido
- [x] blog
- [x] yt_extension_template
- [x] fido-prhandler
- [x] workshop2016
- [x] yt-3.0-paper
- [x] yt-docker
- [x] grid_data_format

These will be skipped for now:

- yt_conda
- yt-asv
- bootcamp2012
- workshop2012
- workshop2013
- yt-supplemental

**Comment by Xarthisius** [comment]
A lot of those are involved in autodeployment / testing. I'm afraid I need to tackle those

**Comment by matthewturk** [comment]
@Xarthisius looks like you started this.  Want to divide up the repos?  I'd be happy to migrate the ones that are obvious.

**Comment by Xarthisius** [comment]
`fido`, `blog`, `website` need simultaneous changes to jenkins. I'll take care of those. The rest is up for grabs.

**Comment by brittonsmith** [comment]
Out of curiosity, is there a git equivalent to "hg convert" that will let you export the history for certain files?  I've used this for some of the analysis modules already, but if there isn't anything on the git side that can do this, maybe I should do it for the others sooner rather than later.

**Comment by ngoldbaum** [comment]
@brittonsmith you can do that with `hg convert` using a filemap.

https://www.mercurial-scm.org/wiki/ConvertExtension#A--filemap

**Comment by ngoldbaum** [comment]
Similarly, it looks like you can use `git filter-branch` to do this once the repo is converted to git: https://help.github.com/articles/splitting-a-subfolder-out-into-a-new-repository/

**Comment by brittonsmith** [comment]
Yes, I know how to use hg convert, which is what I said above.  Was asking about a git equivalent.

**Comment by brittonsmith** [comment]
Was mainly trying to figure out if there is urgency to extract the remaining analysis modules before the move to github is finished.  But, if there's a way in git to do it that's just as easy, this can wait.  Thanks for tip on `git filter-branch`!

**Comment by matthewturk** [comment]
When I used the import functionality, it imported and gave me the option to "map" people between commits.  But, when I try to do this, I have been unable to *actually* make it do the mapping.  The UI seems odd, and I can't quite figure it out.

https://github.com/yt-project/yt_extension_template/import/authors
https://github.com/yt-project/ythub-docs/import/authors

@Xarthisius or @ngoldbaum can you take a look before I import any others?

**Comment by matthewturk** [comment]
Forget I said that!  Fixed.

@Xarthisius can we move the website repo easily, or will it require help too?

I think the YTEP repo should be left in place until at least the GH YTEP PR and the Demeshening PR are accepted.

**Comment by Xarthisius** [comment]
@MatthewTurk the import of the website *repo* is pretty straightforward, but website itself uses `hg churn` I think that needs to be addressed beforehand.

**Comment by ngoldbaum** [comment]
It looks like we'll need a `.mailmap` file: https://git-scm.com/docs/git-shortlog#_mapping_authors

Once we have that we'll be able to parse `git shortlog`.

I'll experiment with this.

**Comment by ngoldbaum** [comment]
Here's a PR to update the website repo so that it gets contributor names from git: https://bitbucket.org/yt_analysis/website/pull-requests/141
